versions/3.0.md

```diff
@@ -71,7 +71,7 @@ The schema exposes two types of fields. Fixed fields, which have a declared name
 
 The OAS representation of the API is made of a single file. However, parts of the definitions can be split into separate files, at the discretion of the user. This is applicable for `$ref` fields in the specification as follows from the [JSON Schema](http://json-schema.org) definitions.
 
-By convention, the OpenAPI Specification (OAS) file is named `swagger.json`.
+By convention, the OpenAPI Specification (OAS) file is named `oas.json`.
```

Comment:
I prefer openapi.* if we must suggest something, although I prefer if we didn't express a preference for a filename.